tests/network/vmi_multus.go

```diff
@@ -333,57 +346,64 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			})
 		})
 
-		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			It("[test_id:1577]should create two virtual machines with one interface", func() {
-				By("checking virtual machine instance can ping the secondary virtual machine instance using Linux bridge CNI plugin")
-				interfaces := []v1.Interface{linuxBridgeInterface}
-				networks := []v1.Network{linuxBridgeNetwork}
-
-				vmiOne := createVMIOnNode(interfaces, networks)
-				vmiTwo := createVMIOnNode(interfaces, networks)
+		Context("With two virtual machines", func() {
+			var (
+				interfaces []v1.Interface
+				networks   []v1.Network
+				vmiOne     *v1.VirtualMachineInstance
+				vmiTwo     *v1.VirtualMachineInstance
+			)
+			JustBeforeEach(func() {
+				vmiOne = createVMIOnNode(interfaces, networks)
+				vmiTwo = createVMIOnNode(interfaces, networks)
 
 				tests.WaitUntilVMIReady(vmiOne, console.LoginToAlpine)
 				tests.WaitUntilVMIReady(vmiTwo, console.LoginToAlpine)
 
-				Expect(configInterface(vmiOne, "eth0", "10.1.1.1/24")).To(Succeed())
-				By("checking virtual machine interface eth0 state")
-				Expect(checkInterface(vmiOne, "eth0")).To(Succeed())
-
-				Expect(configInterface(vmiTwo, "eth0", "10.1.1.2/24")).To(Succeed())
-				By("checking virtual machine interface eth0 state")
-				Expect(checkInterface(vmiTwo, "eth0")).To(Succeed())
-
-				By("ping between virtual machines")
-				Expect(libnet.PingFromVMConsole(vmiOne, "10.1.1.2")).To(Succeed())
 			})
+			Context("With one interface", func() {
+				BeforeEach(func() {
+					interfaces = []v1.Interface{linuxBridgeInterface}
+					networks = []v1.Network{linuxBridgeNetwork}
+				})
+				It("[test_id:1577]should run ping between two virtual machines", func() {
+					Expect(configInterface(vmiOne, "eth0", fmt.Sprintf("%s/24", ptpSubnetIP1))).To(Succeed())
+					By("checking virtual machine interface eth0 state")
+					Expect(checkInterface(vmiOne, "eth0")).To(Succeed())
+
+					Expect(configInterface(vmiTwo, "eth0", fmt.Sprintf("%s/24", ptpSubnetIP2))).To(Succeed())
+					By("checking virtual machine interface eth0 state")
+					Expect(checkInterface(vmiTwo, "eth0")).To(Succeed())
+
+					fmt.Printf("VMs are ready, waiting before ping")
+					time.Sleep(200 * time.Second)
+					By("running ping between virtual machines")
+					Expect(libnet.PingFromVMConsole(vmiOne, ptpSubnetIP2)).To(Succeed(), "ping should succeed")
+				})
+			})
+			Context("With two interfaces", func() {
+				BeforeEach(func() {
+					interfaces = []v1.Interface{
+						defaultInterface,
+						linuxBridgeInterface,
+					}
+					networks = []v1.Network{
+						defaultNetwork,
+						linuxBridgeNetwork,
+					}
+				})
```

Comment:
Does it affect the VM's on the tests specs in this context?

Reply:
same thing here.